### PR TITLE
OS and resolution fixes for Haiku

### PR DESCRIPTION
**Pull request by extrowerk**
## Description

Print the hrev only once.
Fix the resolution

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -2660,7 +2660,7 @@ get_resolution() {
         ;;
 
         "Haiku")
-            resolution="$(screenmode | awk -F ' |, ' '{printf $2 "x" $3 " @ " $6 $7}')"
+            resolution="$(screenmode | tail -1 | awk -F ' |, ' '{printf $2 "x" $3 " @ " $6 $7}')"
```
Comment (dylanaraps) [inline neofetch]: Does this work?

```sh
resolution="$(screenmode | awk -F ' |, ' 'END{printf $2 "x" $3 " @ " $6 $7}')"
```

**Comment by extrowerk** [comment]
Yes, that would work too. It results "1920x1200 @ 60Hz" for me.

black <notifications@github.com>:

**Comment by dylanaraps** [comment]
OK. Make the change and I'll merge. :+1:

**Comment by dylanaraps** [comment]
Fixed this in master.

**Pull request closed without merging**